src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -96,7 +98,13 @@ object ScanProcessor extends CommandProcessor with RuleProcessor {
       config.forceLanguage
     }
     MetricHandler.metricsData("language") = Json.fromString(languageDetected.toString)
-
+    val dependencies: List[DependencyInfo] = config.externalConfigPath.headOption match {
+      case Some(externalConfigPath) =>
+        parseDependencyInfo(
+          List(externalConfigPath, "config", "dependencyInfo", "dependencyinfo.json").mkString(JFile.separator)
+        )
+      case None => List()
+    }
```

Comment:
The only thing needed here is `config.externalConfigPath`. Can we create a method in `DependencyTaggingProcessor` as this piece of code is not need to be handled here

Reply:
Also can we use camelCase for `dependencyinfo.json` as well. To make it consistent

Reply:
done